### PR TITLE
Adding Work Allocation team

### PR DESCRIPTION
**Pull request by raghera**
Adding Work Allocation team to LD

### CODE REVIEW

File: terraform.tfvars [hunk 1]
```diff
@@ -39,5 +39,9 @@ projects = [
   {
     name          = "unspec"
     azuread_group = "40c33f5a-24d0-4b22-a923-df8a80a59cd9"
+  },
+  {
+    name          = "Work-Allocation"
+    azuread_group = "609f67ab-8bab-42cb-910c-b2a61dd5e657"
```
Comment (timja) [inline terraform.tfvars:45]: this should just be your regular team group

Comment (raghera) [reply terraform.tfvars:45]: Ah this one? 
dcd_group_wa_v2                              cdeb331b-adfe-46a7-a2c8-a628e2d35d96

will update

Comment (raghera) [reply terraform.tfvars:45]: I notice some of the other teams e.g. sscs are using their ld group obj id which is why I used the one from `dcd_group_ld_wa_v2` above.

Comment (timja) [reply terraform.tfvars]: yes, that's legacy. the docs should all say team group now

**Review by timja — APPROVED** [review]

**Pull request merged**